mongodb-advanced/index.js

```diff
@@ -40,7 +40,7 @@ function addRoutes(app) {
       if(err) {
         return next(err);
       }
-      res.status(201).end();
+      res.status(200);
```

Comment:
@jeiholzer please re-read my comment that explains that `sendStatus` should be used here: https://github.com/digitalbazaar/bedrock-examples/pull/5#discussion_r148662484